grpc/v1/webserver.proto

```diff
@@ -24,17 +24,13 @@ message SearchRequest {
 }
 
 message SearchResponse {
+  reserved 4, 5;
```

Comment:
I removed this fields following this comment: https://github.com/sourcegraph/zoekt/pull/636#discussion_r1291610295